### PR TITLE
Docs brain immunity/gayle update

### PR DESCRIPTION
**Pull request by GayleNeumann**

### CODE REVIEW

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 1]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
-
-#### Kong Service Map
-
-* For information about the Kong Service Map, see **Using Kong’s Service Map**.
+When you purchase Kong Brain and/or Kong Immunity, you obtain both add-ons together or individually:
+* Kong Brain and Kong Immunity
+* Kong Brain
+* Kong Immunity
 
+![Kong Brain and Kong Immunity] (https://doc-assets.konghq.com/1.3/brain_immunity/KongBrainImmunity_overview.png)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
![Kong Brain and Kong Immunity](https://doc-assets.konghq.com/1.3/brain_immunity/KongBrainImmunity_overview.png)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 2]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
-
-* Kong Immunity is optional.
-* Once the Collector is configured, Kong Immunity is automatically enabled. 
-* View Kong Immunity alerts in the Kong Service Map and take action with just a few clicks.
-* Instantly diagnose issues. Receive Slack alerts notifying users of anomalies that require attention. 
-
-#### Kong Service Map
-
-* For information about the Kong Service Map, see **Using Kong’s Service Map**.
+When you purchase Kong Brain and/or Kong Immunity, you obtain both add-ons together or individually:
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: I thought this was no longer the case?

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 3]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+* Kong Brain
+* Kong Immunity
 
+![Kong Brain and Kong Immunity] (https://doc-assets.konghq.com/1.3/brain_immunity/KongBrainImmunity_overview.png)
 
-### Configure the Kong Collector Plugin
+This guide provides information about how to install, configure, and use Kong Brain and/or Kong Immunity and its components on Kong Enterprise. Sections in this guide include:
 
-#### Prerequisites
+- [Overview] (#overview)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Overview](#overview)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 4]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+- [Prerequisites] (#prerequisites)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Prerequisites](#prerequisites)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 5]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+- [Configure the Collector App and Collector Plugin] (#configure-the-collector-app-and-collector-plugin)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Configure the Collector App and Collector Plugin] #configure-the-collector-app-and-collector-plugin)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 6]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+- [Monitor the Collector] (#monitor-the-collector)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Monitor the Collector](#monitor-the-collector)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 7]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+- [Using Kong Brain] (#using-kong-brain)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Using Kong Brain](#using-kong-brain)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 8]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+- [Using Kong Immunity] (#using-kong-immunity)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Using Kong Immunity](#using-kong-immunity)
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 9]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
 
-Prerequisites for installing the Collector include:
-
-* A working Kong Enterprise system using Kong Enterprise 0.35.3+ or later, with a minimum of one Kong node and a working data store (PostgreSQL or Cassandra).
-* Access to a platform for the Collector which has Docker installed. This system must be networked to “talk” to the Kong Enterprise system on which the Collector is installed.
-* Bintray access credentials (supplied by Kong) to access the downloads.
-
-#### Overview
+### Overview
+Kong Brain and Kong Immunity are installed as add-ons on Kong Enterprise, using a Collector App and a Collector Plugin to communicate with Kong Enterprise. The diagram illustrates how the Kong components work together, and are described below:
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: This mentions a diagram but there is no diagram?

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 10]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+### Configure the Collector App and Collector Plugin
+To enable Kong Brain (Brain) and/or Kong Immunity (Immunity), you must first configure the Collector App and the Collector Plugin. This includes:
+* Deploying the Collector Plugin, which captures and sends traffic to the Collector App for data collection and processing.
+* Deploying the Collector App on your Docker aware platform.
+* Configuring the Collector Plugin and the Collector App to talk to each other. 
+* Testing the configuration to confirm everything is up and running. 
 
-Configure the Collector in the following order, with configuration steps provided below:
+Steps are:
+- [Step 1. Enable the Collector Plugin] (#Step-1.-Enable-the-Collector-Plugin)
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
- [Step 1. Enable the Collector Plugin](#Step-1.-Enable-the-Collector-Plugin)
```

Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: For internal linking to work, there cannot be a space between the `[]` and `()` and the link inside the parenthesis must match the header text, in URL format ie:
`- [Step 1. Enable the Collector Plugin](#step-1-set-up-the-collector-plugin)`

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 11]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+#### Step 1. Set up the Collector Plugin 
+Enable the Collector Plugin using the Admin API:
 
-#### Deploy the Collector 
+```
+$ http --form POST http://<KONG_HOST>:8001/<workspace>/plugins name=collector config.service_token=foo config.host=<COLLECTOR_HOST> config.port=<COLLECTOR_PORT> config.https=false config.log_bodies=true
+```
 
-Deploy the Collector Plugin from the Kong Manager or the Admin API:
+(Optional) It is possible to set up the Collector Plugin to only be applied to specific routes or services, by adding route.id=<ROUTE_ID> or service.id=<SERVICE_ID> to the request.
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
>(Optional) It is possible to set up the Collector Plugin to only be applied to specific routes or services, by adding `route.id=<ROUTE_ID>` or `service.id=<SERVICE_ID>` to the request.
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 12]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+1. Log in to **Bintray** and retrieve your BINTRAY_USERNAME and BINTRAY_API_KEY. 
+2. Click your **username** to get the dropdown menu. 
+3. Click **Edit Profile** to get your BINTRAY_USERNAME.
+4. Click **API Key** to get your BINTRAY_API_KEY. 
 
-### Configure the Collector
-
-#### Bintray Credentials
+#### Step 3: Set up with Docker Compose 
+**Note**: Using Docker Compose is recommended to deploy Brain and Immunity, as documented in this guide. You can also use Docker Swarm or Kubernetes for deployment, although steps are not currently included in this guide.
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
>Note: Using Docker Compose is recommended to deploy Brain and Immunity, as documented in this guide. You can also use Docker Swarm or Kubernetes for deployment, although steps are not currently included in this guide.
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 13]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
-```
-`$ KONG_HOST=<KONG HOST> KONG_PORT=<8001> SQLALCHEMY_DATABASE_URI=<postgres://kong@localhost:5432/collector> docker-compose -f docker-compose.yml -f with-redis.yml up --remove-orphans`
-```
+If you want to store body data in the Collector App, you can do so by setting the Collector’s REDACT_BODY_DATA by updating the environment variable in docker-compose.yml or declaring it in your docker-compose up command as follows:
 
 ```
-`$ REDIS_URI=<redis://localhost:6379/> KONG_HOST=<KONG HOST> KONG_PORT=<8001> docker-compose -f docker-compose.yml -f with-db.yml up --remove-orphans`
+$ REDACT_BODY_DATA=False docker-compose -f docker-compose.yml -f with-redis.yml up
 ```
+#### Step 5. (Optional) Using a different Redis instance
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
#### Step 5. Using a different Redis instance _(optional)_
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 14]
```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
...
+To use your own instance of Redis instead of the one provided by the container, change the command to use your database, Redis, or both.
 
 ```
-`$ REDIS_URI=<redis://localhost:6379/> KONG_HOST=<KONG HOST> KONG_PORT=<8001> SQLALCHEMY_DATABASE_URI=<postgres://kong@localhost:5432/collector> docker-compose -f docker-compose.yml up --remove-orphans`
+$ REDIS_URI=<redis://localhost:6379/> KONG_HOST=<KONG HOST> KONG_PORT=<8001> docker-compose -f docker-compose.yml up
 ```
 
+**Note**: Remove all instances of ```--remove orphans``` and ```--remove orphans a```
```
Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
**Note**: Remove all instances of `--remove orphans` and `--remove orphans a`
```

Comment (levfishbluefish) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion

```

Comment (levfishbluefish) [reply app/enterprise/1.3-x/brain-immunity/install-configure.md]: (It looks like this was just an internal note to update the docs)

**Comment by levfishbluefish** [comment]
Might be outside the scope of this PR, but I still think this should be split up into a series of guides vs one long page

File: app/enterprise/1.3-x/brain-immunity/install-configure.md (hunk 1 above)
Comment (GayleNeumann) [reply app/enterprise/1.3-x/brain-immunity/install-configure.md]: The image goes further down in the docs.

**Pull request merged**